### PR TITLE
omnitrace: add version 6.2.0 and 6.2.1

### PR DESCRIPTION
**Pull request by afzpatel**

### CODE REVIEW

**Review by srekolam — APPROVED** [review]

**Review by renjithravindrankannath — APPROVED** [review]

**Comment by jrmadsen** [comment]
Please do not merge this yet.

**Comment by afzpatel** [comment]
This pr can be merged after @jrmadsen or @dgaliffiAMD have reviewed and approved

**Review by bernhardkaindl — DISMISSED** [review]
@afzpatel: 
> This pr can be merged after @jrmadsen or @dgaliffiAMD have reviewed and approved

The way to make the status on that visible is using the Draft status: It shows in the PR list,
and there is also a label to make this more visible.

File: var/spack/repos/builtin/packages/omnitrace/package.py
Comment (dgaliffiAMD) [inline var/spack/repos/builtin/packages/omnitrace/package.py]:
```suggestion
    homepage = "https://rocm.docs.amd.com/projects/omnitrace/en/latest/index.html"
```
Updating docs page

File: var/spack/repos/builtin/packages/omnitrace/package.py
Comment (dgaliffiAMD) [inline var/spack/repos/builtin/packages/omnitrace/package.py]: I don't think that you  need to include the "main" branch anymore. The default branch is now "amd-staging" and the older commit hashes will also be in this branch.

Comment (afzpatel) [reply var/spack/repos/builtin/packages/omnitrace/package.py]: Ok, I removed it, I also deprecated the older versions of omnitrace

**Review by dgaliffiAMD — APPROVED** [review]

**Comment by bernhardkaindl** [comment]
@spackbot run pipeline

**Review by bernhardkaindl — APPROVED** [review]
> This pr can be merged after @jrmadsen or @dgaliffiAMD have reviewed and approved

@dgaliffiAMD approved and GitLab CI has passed. Will merge now.

**Pull request merged**
